### DISCUSSION TITLE
Question About Integrating Permeability in Two-Phase Liquid-Sediment Simulations

### DISCUSSION THREAD

**Discussion by lucven** [DualSPHysics/DualSPHysics#208] [DUALSPHYSICS_V5.2]
Hello,

I have reviewed the example "Two-phase liquid-sediment implementation in DualSPHysics" in the documentation and noticed that permeability is not considered. I would like to know how to incorporate permeability into the simulation, and what command should be used for this purpose.

Additionally, I would like to know if it is possible to use DesignSPHysics to define soil and its characteristics such as permeability, shear resistance angle, cohesion, etc.

Thank you for your assistance.

**Comment by alexbexe** [comment]
Permeability is not implemented in DualSPHysics. Some works include an Eulerian implementation of permeability or porosity. You can find some examples in the literature. Try to look for "porosity+SPH" in the databases.

Regards
